client/src/main/resources/process-0.1.xsd

```diff
@@ -218,6 +217,7 @@
         </xs:annotation>
         <xs:sequence>
             <xs:element type="validity" name="validity"/>
+            <xs:element type="sla" name="sla" minOccurs="0" maxOccurs="1"/>
```

Comment:
Can a user specify sla at the cluster level only and not specify at the process level? The current xsd allows it. If yes, the parser has to be updated accordingly and also the MonitoringService class. If no, the parser should throw an error.